resources/extruders/creasee_extruder_0.def.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "creasee",
```

Comment:
Moving these files to their correct locations makes the automated tests run on them, to find mistakes in the profile. And the tests pointed out that this field is wrong.

It says here that this is an extruder for the `creasee` printer, but there is no `creasee` printer. You'd need to specify either `creasee_cs20` or `creasee_cs30`. That means that you'll have to duplicate this file as well, since you can only refer to one of the two.

Reply:
> Moving these files to their correct locations makes the automated tests run on them, to find mistakes in the profile. And the tests pointed out that this field is wrong.
> 
> It says here that this is an extruder for the `creasee` printer, but there is no `creasee` printer. You'd need to specify either `creasee_cs20` or `creasee_cs30`. That means that you'll have to duplicate this file as well, since you can only refer to one of the two.

Hi,

ok. I have change that, and I have created another extrueder file. i think now its ok: https://github.com/Ultimaker/Cura/pull/9907/commits